github-sync/github-data/sigstore/repositories.yaml

```diff
@@ -1410,6 +1410,67 @@ repositories:
         dismissalRestrictions:
           - tuf-root-signing-codeowners
           - sigstore-keyholders
+  - name: root-signing-staging
+    owner: sigstore
+    description: "Staging TUF repository for Sigstore trust root"
+    homepageUrl: ""
+    defaultBranch: main
+    allowAutoMerge: false
+    allowMergeCommit: true
+    allowRebaseMerge: false
+    allowSquashMerge: false
```

Comment:
Do you want to allow squash merges? I typically disable merge commits to keep the history clean, allow squash, and allow auto merge.

Reply:
Not 100% sure but I don't think I do...commits in signing event PRs are meaningful as they come from multiple sources(different signers, repository workflow, etc). Merging them would look confusing IMO